docs/tutorials/forum-application/3.send-message.md

```diff
@@ -0,0 +1,382 @@
+# Send Message
+
+**In this section you will learn how a user can send a message on the Forum Application.**
+
+## CheckTx
+
+In ABCI, the `CheckTx` method is used to check the validity of an individual transaction before it is included in the mempool.
+Every node runs CheckTx before letting a transaction into its local mempool.
+
+The `CheckTx` method is responsible for performing any necessary validation checks on the transaction, such as verifying
+the signature, checking for double spending, or enforcing application-specific rules. It is a lightweight and fast
+operation, as it is meant to quickly determine whether a transaction is valid or not.
+
+The method takes in a `RequestCheckTx` object, which contains the transaction to be checked (`req.Tx`) and any other
```

Comment:
You are using `Request*` and `Response*` at several places in this tutorial. However, as of `v1.x` this has changed to `*Request` and `*Response` (so, suffixes rather than prefixes)